### PR TITLE
Add Flaubert OnnxConfig to Transformers

### PR DESCRIPTION
**Pull request by chainyo**
# What does this PR do?

Add Flaubert OnnxConfig to make this model available for conversion.

## Who can review?

Models:
- albert, bert, xlm: @LysandreJik

### CODE REVIEW

File: src/transformers/onnx/features.py [hunk 1]
```diff
@@ -179,6 +180,16 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=DistilBertOnnxConfig,
         ),
+        "flaubert": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
+            "sequence-classification",
+            # "multiple-choice",
```
Comment (lewtun) [inline src/transformers/onnx/features.py:188]: Ah this is some dead code from previous work on the ONNX API, so please remove it. Although Flaubert has a multiple-choice head, we still need to do some work to provide ONNX support for this general class of models.

**Review by lewtun — COMMENTED** [review]
Thanks a lot for adding support for this model @ChainYo! 

It looks like the CI tests are failing because `FlaubertOnnxConfig` cannot be imported. This is a bit weird because the implementation looks OK, but can you check that the slow tests pass locally with:

```
RUN_SLOW=1 python -m pytest -x tests/onnx/test_onnx_v2.py -k "flaubert"
```

File: src/transformers/onnx/features.py (hunk 1 above)
Comment (ChainYo) [reply src/transformers/onnx/features.py:188]: Should I remove this commented line for all other models ?

**Comment by ChainYo** [comment]
> Thanks a lot for adding support for this model @ChainYo!
> 
> It looks like the CI tests are failing because `FlaubertOnnxConfig` cannot be imported. This is a bit weird because the implementation looks OK, but can you check that the slow tests pass locally with:
> 
> ```
> RUN_SLOW=1 python -m pytest -x tests/onnx/test_onnx_v2.py -k "flaubert"
> ```

It seems that it's exactly the same problem locally

```bash
(transformers) chainyo@workstation:~/code/transformers$ pytest -x tests/onnx/test_onnx_v2.py -k "flaubert"
======================================= test session starts =======================================
platform linux -- Python 3.8.12, pytest-7.1.1, pluggy-1.0.0
rootdir: /home/chainyo/code/transformers, configfile: setup.cfg
plugins: timeout-2.1.0, xdist-2.5.0, dash-2.3.0, hypothesis-6.39.4, forked-1.4.0
collected 0 items / 1 error                                                                       

============================================= ERRORS ==============================================
___________________________ ERROR collecting tests/onnx/test_onnx_v2.py ___________________________
ImportError while importing test module '/home/chainyo/code/transformers/tests/onnx/test_onnx_v2.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../../miniconda3/envs/transformers/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/onnx/test_onnx_v2.py:21: in <module>
    from transformers.onnx.features import FeaturesManager
src/transformers/onnx/features.py:11: in <module>
    from ..models.flaubert import FlaubertOnnxConfig
E   ImportError: cannot import name 'FlaubertOnnxConfig' from 'transformers.models.flaubert' (/home/chainyo/code/transformers/src/transformers/models/flaubert/__init__.py)
----------------------------------------- Captured stderr -----------------------------------------
2022-03-21 19:30:32.268001: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2022-03-21 19:30:32.268048: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
WARNING:absl:No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
======================================== warnings summary =========================================
../../miniconda3/envs/transformers/lib/python3.8/site-packages/flatbuffers/compat.py:19
  /home/chainyo/miniconda3/envs/transformers/lib/python3.8/site-packages/flatbuffers/compat.py:19: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
===================================== short test summary info =====================================
ERROR tests/onnx/test_onnx_v2.py
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
=================================== 1 warning, 1 error in 0.29s ===================================
```

**Comment by ChainYo** [comment]
Ok I got it, it was a problem of indentation... :+1:

File: src/transformers/onnx/features.py (hunk 1 above)
Comment (lewtun) [reply src/transformers/onnx/features.py]: Thanks for the offer, but no let's do that in a separate PR to keep this one focused on just FlauBERT :)

**Review by lewtun — APPROVED** [review]
Thanks for iterating on this @ChainYo! The PR LGTM and gently pinging @LysandreJik or @sgugger for final approval

**Review by sgugger — APPROVED** [review]
Thanks for adding support on this model!

**Pull request merged**